apps/src/templates/manageStudents/manageStudentsRedux.js

```diff
@@ -209,6 +235,34 @@ export const addMultipleAddRows = (studentNames) => {
   };
 };
 
+export const transferStudents = () => {
+  return (dispatch, getState) => {
+    const state = getState();
+    const currentSectionCode = sectionCode(state, state.manageStudents.sectionId);
```

Comment:
Given our original concern, might be good to have a comment here pointing out we are pulling information from another part of the redux state tree.